go/webhook.go

```diff
@@ -96,19 +117,21 @@ func (wh *Webhook) Sign(msgId string, timestamp time.Time, payload []byte) (stri
 
 }
 
-func verifyTimestamp(timestampHeader string) (time.Time, error) {
+func verifyTimestamp(timestampHeader string, enforceTolerance bool) (time.Time, error) {
```

Comment:
Isn't this whole function used to verify the tolerance? I think it shouldn't be a parameter to this, just an if statement whether to call this one or not, no?

Reply:
it also was parsing the timestamp to a time.Time but pulled this out to a separate function.